api/camunda/camunda-openapi.yaml

```diff
@@ -983,6 +1355,123 @@ components:
           format: int64
         tenantId:
           type: string
+    IncidentSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/IncidentFilterRequest"
+    IncidentFilterRequest:
+      type: object
+      properties:
+        key:
+          type: integer
+          format: int64
+        processDefinitionKey:
+          type: integer
+          format: int64
+        processInstanceKey:
+          type: integer
+          format: int64
+        type:
+          type: string
+        message:
+          type: string
+        flowNodeId:
+          type: string
+        flowNodeInstanceId:
+          type: string
+        state:
+          type: string
+        jobKey:
+          type: integer
+          format: int64
+        tenantId:
+          type: string
+        hasActiveOperation:
+          type: boolean
```

Comment:
change-at-the-source, descriptions, new:

These properties all need descriptions. We should follow up with the original author of this endpoint to add them.

Reply:
> These properties all need descriptions. We should follow up with the original author of this endpoint to add them.

I agree 👍

Reply:
@ralfpuchert - in [a recent PR](https://github.com/camunda/camunda/pull/21414/) to the C8 REST API, you added the following, each of which is missing `description` properties in the OpenAPI spec:

- the [POST /incidents/search](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1113-L1114) endpoint
- [IncidentFilterRequest](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1366) schema
- [IncidentItem](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1404) schema
- [OperationItem](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1441) schema
- [ProcessInstanceReferenceItem](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1457) schema
- [DecisionInstanceReferenceItem](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml#L1467) schema

The descriptions are important because the docs ingest the spec from the `camunda` repo, and generate our API docs from it. I'm tagging you here in the `camunda-docs` repo, instead of the `camunda` repo, because that's where we (DevEx) noticed the descriptions missing.

Please add descriptions to these endpoints/properties, in the [C8 REST OpenAPI spec](https://github.com/camunda/camunda-docs/blob/main/api/camunda/camunda-openapi.yaml). Your changes will eventually propagate downstream to the docs here.

For guidance on writing the descriptions, [see here](https://github.com/camunda/camunda/pull/21999). If you have any questions, feel free to ask. 

Thank you!